zzz_generated/app-common/app-common/zap-generated/cluster-enums.h

```diff
@@ -70,13 +70,13 @@ static IdentifyEffectVariant __attribute__((unused)) kIdentifyEffectVariantkUnkn
 // Enum for IdentifyIdentifyType
 enum class IdentifyIdentifyType : uint8_t
 {
-    kNone         = 0x00,
-    kVisibleLight = 0x01,
-    kVisibleLED   = 0x02,
-    kAudibleBeep  = 0x03,
-    kDisplay      = 0x04,
-    kActuator     = 0x05,
-    // kUnknownEnumValue = 6,
+    kNone             = 0x00,
+    kVisibleLight     = 0x01,
+    kVisibleLED       = 0x02,
+    kAudibleBeep      = 0x03,
+    kDisplay          = 0x04,
+    kActuator         = 0x05,
+    kUnknownEnumValue = 6,
```

Comment:
@jmeg-sfy : I am just a bot :)( (well, this PR is) - this is just a cherry pick, of already merged changes... Can you raise this issue?

Reply:
@jmeg-sfy I actually asked that same question.  ;)

For code written _before_ that spec update, the 6 is an unknown value and will get treated as unknown value.  

For code written after the spec update, it will select a different value to use as unknown value.

This all works as long as no one _sends_ unknown value (which they really should not do).